From 4d2564003fd0e5bf98448afea84746704363abc4 Mon Sep 17 00:00:00 2001
From: Jason Hu <jason.hu@intel.com>
Date: Thu, 16 Aug 2012 15:16:44 -0400
Subject: [PORT FROM MAIN] add a hint to notify surfaceflinger to disable rotation animation

BZ: 50798

DO NOT UPSTREAM

Change-Id: I5ca72b138b064173370f32199934da0e3b2c96fc
Orig-Change-Id: Ic947ded7920ac576302a0f6c7504abbfd34e2bdd
Signed-off-by: Jason Hu <jason.hu@intel.com>
Reviewed-on: http://android.intel.com:8080/75180
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Ding, Haitao <haitao.ding@intel.com>
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
---
 include/hardware/hwcomposer_defs.h |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/include/hardware/hwcomposer_defs.h b/include/hardware/hwcomposer_defs.h
index 1edfd3d..f782ca3 100644
--- a/include/hardware/hwcomposer_defs.h
+++ b/include/hardware/hwcomposer_defs.h
@@ -61,7 +61,13 @@ enum {
      * SurfaceFlinger will only honor this flag when the layer has no blending
      *
      */
-    HWC_HINT_CLEAR_FB       = 0x00000002
+    HWC_HINT_CLEAR_FB       = 0x00000002,
+
+    /*
+     * HWC set HWC_HINT_DISABLE_ANIMATION hint to indicate to SurfaceFlinger
+     * that it should bypass creating screen shot layer for rotation animation.
+     */
+    HWC_HINT_DISABLE_ANIMATION  = 0x00000004
 };
 
 /*
-- 
1.7.4.1

